### PR TITLE
B3 Exports

### PR DESCRIPTION
**Pull request by benrudolph**
@biyeun this converts the new exports to b3 (when they turn the flag on). i also implemented a more performant drag and drop sorting since we've had bugs where people complain they can't sort the columns. i decided to leave that commit out since it was a pretty big change and i'm not sure it's worth the added complexity since it's an edge case. best read 🐟  and with no whitespace

cc: @sravfeyn

### CODE REVIEW

File: corehq/apps/export/static/export/spec/Exports.Utils.spec.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ describe('Export Utility functions', function() {
     describe('#getTagCSSClass', function() {
         it('Should get regular tag class', function() {
             var cls = Exports.Utils.getTagCSSClass('random-tag');
-            assert.equal(cls, 'label');
+            assert.equal(cls, 'label label-default');
```
Comment (dimagimon) [inline corehq/apps/export/static/export/spec/Exports.Utils.spec.js]: 'assert' is not defined. (no-undef)

**Comment by benrudolph** [comment]
And some screen shots:
screen shot 2016-04-19 at 2 32 49 am
screen shot 2016-04-19 at 2 32 43 am

**Comment by biyeun** [comment]
@benrudolph what's the ETA for having this flag get turned off? Would we be waiting on QA?

File: corehq/apps/export/templates/export/bootstrap3/customize_export.html [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+                    {% trans "Export Name" %}
+                </label>
+                <div class="col-sm-4">
+                    <input type="text" class="form-control" id="export-name" data-bind="value: name" />
+                </div>
+            </div>
+            <div class="form-group">
+                <label
+                    for="format-select"
+                    class="col-sm-2 control-label">
```
Comment (biyeun) [inline corehq/apps/export/templates/export/bootstrap3/customize_export.html]: `col-sm-2` seems really small for actual `sm` screens and a lot of these label sizes

recommend: 
for labels `col-xs-4 col-md-3 col-lg-2`
for fields `col-xs-8 col-md-8 col-lg-9`

File: corehq/apps/export/templates/export/partial/new_customize_export_templates.html [hunk 3]
```diff
@@ -14,48 +14,48 @@
...
+                <div class="col-sm-4">
+                    <input type="text" class="form-control" data-bind="value: table.label" />
                 </div>
             </div>
-            <div class="control-group">
-                <label class="control-label">
+            <div class="form-group">
+                <label class="col-sm-2 control-label">
                     <strong>{% trans "Choose the fields you want to export." %}</strong><br />
+                    <span style="font-weight: normal">
```
Comment (biyeun) [inline corehq/apps/export/templates/export/partial/new_customize_export_templates.html]: hmm, this is a lot of text for a field label. better as either help text or question bubble

Comment (benrudolph) [inline corehq/apps/export/templates/export/partial/new_customize_export_templates.html]: this was the previous implementation, happy to change if you think it's better. it doesn't look horrible and we have extra space in that margin so i could go either way

File: corehq/apps/export/templates/export/bootstrap3/customize_export.html (hunk 2 above)
Comment (benrudolph) [inline corehq/apps/export/templates/export/bootstrap3/customize_export.html]: 👍

**Comment by benrudolph** [comment]
ETA for getting the flag turned on is a pretty long while. i'm imagining a couple months because it's bundled with the export backend rewrite

**Comment by biyeun** [comment]
@benrudolph ok, I definitely want to remove bootstrap2 dependencies long before then. any chance this can be updated with the non-flagged template?

File: corehq/apps/export/templates/export/partial/new_customize_export_templates.html (hunk 3 above)
Comment (biyeun) [inline corehq/apps/export/templates/export/partial/new_customize_export_templates.html]: ok, fine to leave if it looks ok, but i would update the field and label sizes.

**Comment by benrudolph** [comment]
ok B i think this is g2g when tests pass. for now i'll hold off on doing the other export page because that'll probably need to go through QA since it's in use

File: corehq/apps/export/templates/export/bootstrap3/customize_export.html (hunk 2 above)
Comment (benrudolph) [inline corehq/apps/export/templates/export/bootstrap3/customize_export.html]: oh wait, need to update fields

**Comment by biyeun** [comment]
:+1: on ✅

**Comment by benrudolph** [comment]
sweet, i'm happy with how it looks, i think the biggest thing was to have it stack for `xs` since the screen is just too small to try and fit things into columns

**Comment by benrudolph** [comment]
✅

**Pull request merged**